app/src/main/res/layout/activity_main.xml

```diff
@@ -1,39 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there any reason why we switched to a `RelativeLayout`? (I'm just a big fan of ConstraintLayouts lol)

Reply:
I guess if its simple enough, it doesnt make sense to do extra work for the Constraint - idk

Reply:
No reason why I changed it was just a mistake. I can change it back if you'd like doesn't matter to me